README.adoc

```diff
@@ -38,15 +38,17 @@ installs the Open Liberty runtime, creates a server, and installs the applicatio
 The application displays a simple web page with a link. When you click that link, the application 
 calls the servlet to return a simple response of `Hello! Is Gradle working for you?`.
 
-One benefit of using a build tool like Gradle is that after you define the details of the 
-project and any dependencies it has, Gradle automatically downloads and installs the dependencies.
-
-Another benefit of using Gradle is that it can run repeatable, automated tests on the application.
-You could, of course, test your application manually by starting a server and pointing a web 
-browser at the application URL. Automated tests are a much better approach because you can 
-easily rerun the same tests each time the application is built. If the tests don't pass after 
-you've made a change to the application, the build fails, and you know that you introduced 
-a regression that requires a fix to your code.
+Using a build tool like Gradle is beneficial as you can define the details of the projects and its dependencies.
```

Comment:
I think a combination of the old and new is clearer: "One benefit of using a build tool like Gradle is that you can define the details of the project and any dependencies it has, and then Gradle automatically downloads and installs the dependencies."